src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void createService() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("YuliaTsariv");
+        user.setAge(20);
+        user.setPassword("Yulichka12");
+    }
+
+    @Test
+    void register_withMinAge_ok() {
+        user.setAge(18);
+        registrationService.register(user);
+        assertEquals(user, storageDao.get(user.getLogin()));
+    }
+
+    @Test
+    void register_withMinPassword_length_ok() {
+        user.setPassword("mypass");
+        registrationService.register(user);
+        assertEquals(user, storageDao.get(user.getLogin()));
+    }
+
+    @Test
+    void valid_allData_ok() {
+        registrationService.register(user);
+        assertEquals(user, storageDao.get(user.getLogin()));
+    }
```

Comment:
I think we can combine these methods
also in previous solution we tested returned value
<img width="420" alt="image" src="https://user-images.githubusercontent.com/58032907/197964689-93396757-a813-4deb-9fd3-d7c2bb29f300.png">
so let's write one method that register user successfully, and check returned value and user in storage